javascript-loops-practice/sum-all.js

```diff
@@ -0,0 +1,10 @@
+/* exported sumAll */
+function sumAll(numbers) {
+  var i = numbers.length;
+  var sum = 0;
+  while (i > 0) {
+    sum += numbers[i - 1];
+    i--;
+  }
```

Comment:
Same feedback here, there's no real reason this loop needs to be going backwards, please make it move forwards through the array.